codegen/rustjit.rs

```diff
@@ -17,16 +17,30 @@ pub struct RustjitContext {
 impl RustjitContext {
     pub fn get(&self, value: IrValue) -> u64 {
```

Comment:
I think it is better introduce a intermediate(name example: `RustjitIV`) value than using u64 here.

Reply:
I'll try